epochStart/bootstrap/process_test.go

```diff
@@ -161,6 +162,12 @@ func TestIsStartInEpochZero(t *testing.T) {
 func TestEpochStartBootstrap_BootstrapStartInEpochNotEnabled(t *testing.T) {
 	args := createMockEpochStartBootstrapArgs()
 
+	err := errors.New("localErr")
+	args.LatestStorageDataProvider = &mock.LatestStorageDataProviderStub{
+		GetCalled: func() (storage.LatestDataFromStorage, error) {
+			return storage.LatestDataFromStorage{}, err
```

Comment:
Is this intended? Returning both useful data and error?

Reply:
It is not useful. It is a strict not a pointer to a struct. Only error is important for this

Reply:
yup, is not a pointer, missed that